app/presenters/summary/sections/justification_for_legal_aid.rb

```diff
@@ -0,0 +1,36 @@
+module Summary
+  module Sections
+    class JustificationForLegalAid < Sections::BaseSection
+      def name
+        :justification_for_legal_aid
+      end
+
+      def show?
+        kase.present? && super
+      end
+
+      def answers
+        [
+          ioj.types.map do |ioj_type|
+            justification = "#{ioj_type}_justification"
```

Comment:
It is best if we keep using the `#justification_field_name` method. We can instantiate the value-object with `IojReasonType.new(ioj_type)` and then call the method.